TFT/src/User/Menu/PowerFailed.c

```diff
@@ -125,21 +125,22 @@ bool powerOffGetData(void)
   if (f_read(&fp, &infoBreakPoint,  sizeof(infoBreakPoint), &br)   != FR_OK) return false;
 
   setPrintModelIcon(model_icon);
-  for (int8_t i = MAX_HEATER_COUNT - 1; i >= 0; i--)
-  {
-    if (infoBreakPoint.target[i] != 0)
-      mustStoreCacheCmd("%s S%d\n", heatWaitCmd[i], infoBreakPoint.target[i]);
-  }
 
   for (uint8_t i = 0; i < infoSettings.fan_count; i++)
   {
-    if (infoBreakPoint.fan[i] != 0 && fanIsType(i, FAN_TYPE_F))
+    if (infoBreakPoint.fan[i] != 0)
     {
       mustStoreCacheCmd("%s S%d\n", fanCmd[i], infoBreakPoint.fan[i]);
     }
   }
 
   mustStoreCacheCmd("%s\n", tool_change[infoBreakPoint.tool]);
+  for (int8_t i = MAX_HEATER_COUNT - 1; i >= 0; i--)  // infoCacheCmd.queue[0 - 7]
```

Comment:
```suggestion
  for (int8_t i = infoSettings.hotend_count - 1; i >= 0; i--)  // infoCacheCmd.queue[0 - 7]
```

The cause for the Invalid extruder error is that the PLR API is sending temperature command for all hotends (`MAX_HEATER_COUNT`), even if there is only one hotend available. The current logic is sending the commands directly to the cmd cache without checking the actual hotend count reported by the machine. Replacing `MAX_HEATER_COUNT` with `infoSettings.hotend_count` should fix the issue.